TwitchLib.Api/TwitchAPI.cs

```diff
@@ -35,6 +34,40 @@ public TwitchAPI(ILoggerFactory loggerFactory = null, IRateLimiter rateLimiter =
             V5 = new V5.V5(loggerFactory, _rateLimiter, Settings, _http);
             ThirdParty = new ThirdParty.ThirdParty(Settings, _rateLimiter, _http);
             Undocumented = new Undocumented(Settings, _rateLimiter, _http);
+
+            settings.PropertyChanged += Settings_PropertyChanged;
+        }
+
+        private void Settings_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            switch (e.PropertyName)
+            {
+                case "AccessToken":
```

Comment:
Lets use nameof() here instead literal strings! Makes it easier to refactor.

Reply:
Same for the other cases of course

Reply:
yep